app/javascript/react/components/molecules/StationValueLabel/StationValueLabel.style.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components";
+
+const GradientBox = styled.div`
+  background: linear-gradient(
+      241deg,
+      rgba(0, 181, 239, 0.4) -2.4%,
+      rgba(0, 181, 239, 0) 94.94%
```

Comment:
Is there a possibility to make those `rgba(0, 181, 239, 0)` separate variables? Or move to Colors? Just to make it a little bit more readable :)